doc/api/url.md

```diff
@@ -313,20 +313,39 @@ myURL.port = 1234.5678;
 console.log(myURL.port);
 // Prints 1234
 
-// Out-of-range numbers are ignored
-myURL.port = 1e10;
+// Out-of-range numbers, which are not represented in scientific noation,
+// will be ignored.
+myURL.port = 1e10; // 10000000000, will be range-checked as described below
 console.log(myURL.port);
 // Prints 1234
+
 ```
 
 The port value may be set as either a number or as a String containing a number
 in the range `0` to `65535` (inclusive). Setting the value to the default port
 of the `URL` objects given `protocol` will result in the `port` value becoming
 the empty string (`''`).
 
-If an invalid string is assigned to the `port` property, but it begins with a
-number, the leading number is assigned to `port`. Otherwise, or if the number
-lies outside the range denoted above, it is ignored.
+Upon assigning a value to the port, the value will first be converted to a string using `.toString()`.
+
+If that string is invalid, but it begins with a
```

Comment:
Not sure why, but the wrapping of these two paragraphs creates hard wraps in the rendered text: https://github.com/nodeav/node/blob/e402ff4b54dd1eeb0cd0e7ce346751e97769c880/doc/api/url.md#urlport

Maybe we can rewrap them with more long lines (but within 80 characters)?

Reply:
I'm not sure I understand, in which line does the hard wrap occur?
And are we really limited to 80 characters? Existing lines in this doc are considerably longer

Reply:
Yes, existing lines more than 80 chars is a temporal legacy, but now we have a rule in the [STYLE_GUIDE.md](https://github.com/nodejs/node/blob/master/doc/STYLE_GUIDE.md) and doc linting should throw on limit breaking.

I mean these hard
![hb](https://user-images.githubusercontent.com/10393198/38238168-b93b61c0-3731-11e8-8386-409e823a9adf.png)
 breaks:

Reply:
Oh, I see  You have trailing spaces after some added lines, and markdown parses such spaces as hard breaks, It is usually useful to trim trailing spaces in the editor before saving the doc.

Reply:
Oh, the trailing spaces were actually intentional, but I don't mind changing that.

and good to know about the line widths

Reply:
And doc linting makes exceptions for long lines with URLs and code and maybe some other edge cases, so these lines can still be long)

Reply:
Trailing spaces are not usually safe to rely upon in shared sources: some collaborator may edit any other line and have their editor set for trailing spaces auto-trimming in the whole doc, so these spaces may be deleted silently in any other PR)

Reply:
Is there an actual designated docs linter? How do I run it?
(I made the changes manually for now)

by the way, automatically removing trailing spaces for markdown documents is a bug IMO, as it's ignoring a part of the markdown language

Reply:
I made sure the regex `\ $` doesn't find any result in the document.

Reply:
If I am not mistaken. this is how to lint docs locally:
```
make lint-md-build
make test-doc
```

Reply:
Thanks! The first one says there's nothing to be done; the second one passes.